fe/fe-core/src/main/java/org/apache/doris/tablefunction/LocalTableValuedFunction.java

```diff
@@ -86,9 +86,9 @@ private void getFileListFromBackend() throws AnalysisException {
             Future<PGlobResponse> response = proxy.glob(address, requestBuilder.build());
             PGlobResponse globResponse = response.get(5, TimeUnit.SECONDS);
             if (globResponse.getStatus().getStatusCode() != 0) {
-                throw new AnalysisException(
-                        "error code: " + globResponse.getStatus().getStatusCode()
-                                + ", error msg: " + globResponse.getStatus().getErrorMsgsList());
+                LOG.warn("get file from backend failed. error code: {}, error msg: {}",
```

Comment:
I think we should keep this exception.
For local tvf, it mains used for debug of POC, better expose every exception

Reply:
done